portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/remotewebapps/RemoteApps.testcase

```diff
@@ -63,6 +63,28 @@ definition {
 			entryType = "${customElementType}");
 	}
 
+	@description = "Verify that remote app of type Custom Element can be deleted"
+	@priority = "4"
+	test CustomElementCanBeDeleted {
+		property portal.acceptance = "true";
+
+		var customElementName = "Vanilla Counter";
+		var customElementType = "Custom Element";
+		var customElementHTMLName = "vanilla-counter";
+		var customElementURL = "http://remote-component-test.wincent.com/packages/vanilla-counter/index.js";
+
+		RemoteApps.goToRemoteAppsPortlet();
+
+		RemoteApps.addCustomElement(
+			entryHTMLName = "${customElementHTMLName}",
+			entryName = "${customElementName}",
+			entryURL = "${customElementURL}");
```

Comment:
I had mentioned this with @Esther-OC before too and was ok with it at first but now I still don't like this format. We want to prioritize readability higher than reusability and fancy code. It will be better if we put what the values are here instead of setting the variables at the beginning of the test. If the test fails, I have to look up to the beginning steps of the test or read the poshi report details to better understand what was input. This would cause more time to figure out what went wrong.